src/content/perf/ui-performance.md

```diff
@@ -133,12 +125,25 @@ UI performance of your application on a frame-by-frame basis.
 Once your app is running in profile mode,
 [launch DevTools][].
 
-[launch DevTools]: /tools/devtools
-[Timeline view]: /tools/devtools/performance
+## Display the performance overlay {:#displaying-the-performance-overlay}
+
+You can toggle display of the performance overlay as follows:
+
+* **Flutter inspector**: The easiest way to enable the PerformanceOverlay widget
```

Comment:
Just checking. Is this the _new_ inspector or the old one? Maybe it makes no difference.

Reply:
I'm not sure. I think this might be the old one. I'll look into it right now.

Reply:
@elliette can you take a look at this?

Reply:
Hi! This has actually been moved in both the new **and** old inspector. The  Performance Overlay button is in the Performance panel of DevTools, not the Inspector.

Reply:
Updated!